app/cdap/components/WrangleHome/Components/WrangleCard/index.tsx

```diff
@@ -15,146 +15,72 @@
  */
 
 import { Box, Card, Typography } from '@material-ui/core';
-import { fetchConnectors } from 'components/Connections/Create/reducer';
+import { getWidgetData } from 'components/WrangleHome/services/getWidgetData';
 import React, { useEffect, useState } from 'react';
 import { Link } from 'react-router-dom';
 import { getCurrentNamespace } from 'services/NamespaceStore';
-import { BigQuery } from './iconStore/BigQuerySVG';
-import { CloudSQLMySQL } from './iconStore/CloudSQLMySQL';
-import { CloudSQLPostGreSQL } from './iconStore/CloudSQLPostGreSQL';
-import { Database } from './iconStore/Database';
-import { GCS } from './iconStore/GCS';
-import { ImportDatasetIcon } from './iconStore/ImportDatasetIcon';
-import { Kafka } from './iconStore/Kafka';
-import { MySQL } from './iconStore/MySQL';
-import { Oracle } from './iconStore/Oracle';
-import { PostgreSQL } from './iconStore/PostgreSQL';
-import { S3 } from './iconStore/S3';
-import { Spanner } from './iconStore/Spanner';
-import { SQLServer } from './iconStore/SQLServer';
-import { useStyles } from './styles';
-
-export default function WrangleCard() {
-  const [connectorTypes, setConnectorTypes] = useState({
-    fetchedConnectorTypes: [],
-  });
+import { useStyles } from 'components/WrangleHome/Components/WrangleCard/styles';
+import { IConnectorArray } from 'components/WrangleHome/Components/WrangleCard/types';
 
+export default function({ toggleViewAllLink }) {
```

Comment:
add type for toggleViewAllLink

Reply:
added interface in types file and exported it to index.tsx. Done.